b2b-sample/README.md

```diff
@@ -108,10 +105,6 @@ To create a user for Best Auto Mart with permissions to create an IdP for the **
         "name": "Guardio Insurance",
         "tag": "Anytime . Anywhere"
       }
-    },
-    "ManagementAPIConfig": {
-      "SharedApplicationName": "Guardio-Business-App",
-      "UserStore" : "DEFAULT"
```

Comment:
These configs are used by the admin app. Hence keeping these configs within business app can cause confusions. By moving that config to admin app config section will be helpful incase the config.json file was required to be managed as two separate files to configure for the maintainability.